src/components/LandingPage.js

```diff
@@ -33,29 +38,89 @@ class LandingPage extends React.Component {
       this.props.dispatchSetBackendModeSignedInStatusAction(BACKEND_MODE_SIGNED_IN_STATUS.USER_SIGNED_IN);
     }
   };
-	
+
   render = () =>
-    <Hero
-      color="black"
-      bg="white"
-      backgroundImage="https://source.unsplash.com/npxXWgQ33ZQ/1600x900">
-      <Section heading='YADA' subhead='Yet Another Docs App' width={1}>
-        <GoogleLogin
-          clientId={CLIENT_ID}
-          buttonText="Sign in with Google to get started"
-          onSuccess={response => { this.handleLoginSuccess(response); }}
-          onFailure={response => { console.log(response); }}
-          cookiePolicy="single_host_origin"
-          responseType="code,token"
-        />
-      </Section>
-      <button
-        onClick={() => {
-          this.props.dispatchSetBackendModeSignedInStatusAction(BACKEND_MODE_SIGNED_IN_STATUS.LOCAL_STORAGE);
-        }}>
-        Use local storage
-      </button>
-    </Hero>;
+    // <Hero
+    //   color="black"
+    //   bg="white"
+    //   backgroundImage="https://source.unsplash.com/npxXWgQ33ZQ/1600x900">
+    //   <Section heading='YADA' subhead='Yet Another Docs App' width={1}>
+    //     <GoogleLogin
+    //       clientId={CLIENT_ID}
+    //       buttonText="Sign in with Google to get started"
+    //       onSuccess={response => { this.handleLoginSuccess(response); }}
+    //       onFailure={response => { console.log(response); }}
+    //       cookiePolicy="single_host_origin"
+    //       responseType="code,token"
+    //     />
+    //   </Section>
+    //   <button
+    //     onClick={() => {
+    //       this.props.dispatchSetBackendModeSignedInStatusAction(BACKEND_MODE_SIGNED_IN_STATUS.LOCAL_STORAGE);
+    //     }}>
+    //     Use local storage
+    //   </button>
+    // </Hero>;
+    <div style={{display:"flex", flexFlow: "column", height: "100vh", overflow: "hidden"}}>
+      <div className={"landing-root"} style={{flexGrow: "0 1 auto"}}>
+      <AppBar position="static" class="custom-navbar">
+        <Toolbar>
+          <img className={"menuButton"} src={require('../images/logo.png')} style={{width: "50px", marginRight: "1%"}}/>
+          {/* <Typography variant="h5" className={"title"} style={{fontFamily:"Bungee", color:"#F5F0E1"}}>
+            YADA
+          </Typography> */}
+        </Toolbar>
+      </AppBar>
+
+    </div>
+    <div style={{backgroundColor: "#F5F0E1", flex: "1 1 auto"}}>
+    <Grid container spacing={3} alignItems="center">
+        <Grid item xs={6} alignItems="center" style={{marginTop: "10%"}}>
+          <Grid container spacing={3} alignItems="center">
+            <Grid item xs={3}>
```

Comment:
try playing around with `xs`, `sm` and `md`. This will ensure that when the window is resized the logo goes below the buttons